src/overlay/Peer.h

```diff
@@ -55,6 +55,26 @@ class Peer : public std::enable_shared_from_this<Peer>,
   public:
     typedef std::shared_ptr<Peer> pointer;
 
+    // Data structure that records the time it takes to process a message
```

Comment:
I don't think the part of this comment about destructors is true anymore, right?